shared/utility/behaviour/MotionCommand.hpp

```diff
@@ -59,7 +59,19 @@ namespace utility::behaviour {
         return cmd;
     }
 
-    // TODO(BehaviourTeam): Create accessor methods that throw errors if the data
+    inline MotionCommand RotateOnSpot() {
+        MotionCommand cmd;
+        cmd.type = MotionCommand::Type::Value::ROTATE_ON_SPOT;
+        return cmd;
+    }
+
+    inline MotionCommand WalkToReady() {
+        MotionCommand cmd;
+        cmd.type = MotionCommand::Type::Value::WALK_TO_READY;
+        return cmd;
+    }
+
+    // TODO: Create accessor methods that throw errors if the data
```

Comment:
```suggestion
    // TODO(BehaviourTeam): Create accessor methods that throw errors if the data
```

Reply:
If you're going to assign things to teams you should make GitHub teams and assign them things!

Reply:
We do have github teams